cmd/swapd/main.go

```diff
@@ -230,6 +238,7 @@ func setLogLevels(c *cli.Context) error {
 	_ = logging.SetLogLevel("cmd", level)
 	_ = logging.SetLogLevel("net", level)
 	_ = logging.SetLogLevel("rpc", level)
+	_ = logging.SetLogLevel("monero", level)
```

Comment:
I don't know what the solution is, but it would be nice if we came up with a solution that doesn't require updating every time we add or rename a package.  I tried (but reverted) a change that set the log level for `"*"`, but that had to downside of including lots of verbose logs from the libp2p libraries which we may not want.

Reply:
hm, does `github.com/athanorlabs/atomic-swap/*` work as the package name? yeah, definitely don't want those libp2p logs

Reply:
I'm not sure.  I'll create a ticket to look into it.